homework-projects/README.md

```diff
@@ -0,0 +1,59 @@
+# WHAT
+
+During the JS2 and JS3 modules, you will be working on a project instead of your weekly homework assignments. Each week, you will be required to add new features to your project. By the end of JS3, you will have built an actual app using your newly gained knowledge and coding skills!
+<br/>
+
+![](/assets/project-flow.png)
+
+<br/>
+
+# WHY
+
+We're doing this for four main reasons:
+
+- You will gain experience and an understanding of what it's like to work on **one continuous project**, instead of switching contexts with each new homework assignment.
+- You will apply what you learned every week and understand how each new method can be implemented into an actual product.
+- You will build a solid project for your portfolio that you can showcase on your CV and in future tech interviews!
+- You will gain experience presenting your project and code in a 'mini-tech interview' at the end of JS3.
+
+<br/>
+
+# HOW
+
+<br/>
+First, you will pick a project idea (see the "[projects](url)" folder). Each week, you will have specific tasks that you have to deliver. Those tasks are provided in your chosen project description. The tasks are designed to help you practice the main learning points throughout the JS2 and JS3 modules:
```

Comment:
is the `url` here a stub for purpose?

Reply:
Ah yes, good catch - i'll fix it! Thanks!